packages/meditrak-server/src/routes/userEntityPermissions/assertUserEntityPermissionPermissions.js

```diff
@@ -3,45 +3,35 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import {
-  BES_ADMIN_PERMISSION_GROUP,
-  TUPAIA_ADMIN_PANEL_PERMISSION_GROUP,
-} from '../../permissions/constants';
+import { hasBESAdminAccess, TUPAIA_ADMIN_PANEL_PERMISSION_GROUP } from '../../permissions';
 
 export const assertUserEntityPermissionPermissions = async (
   accessPolicy,
   models,
   userEntityPermission,
 ) => {
-  const results = await filterUserEntityPermissionsByPermissions(
-    accessPolicy,
-    [userEntityPermission],
-    models,
-  );
-  if (results.length === 0) {
+  const entity = await models.entity.findById(userEntityPermission.entity_id);
```

Comment:
I've looked into the problem reported by Klaus this morning and I think it is related to this line of code. This might not work for the Admin Panel because it doesn't request for `entity_id` column. So userEntityPermission record does not have `entity_id` value 😢 

![permission](https://user-images.githubusercontent.com/63621318/95029744-25278300-06f6-11eb-9f3b-598bc5408dad.PNG)

Reply:
Good catch! I've swapped out the permission checks to accept the `id` rather than the object. Then we look up the model before running the checks so we know exactly what we're dealing with.

I've swapped this over for the user and permission endpoints that this PR is touching, I'll fix up the rest alongside the rest of the SQL-ized work in another PR.